go/Makefile.am

```diff
@@ -1,2 +1,15 @@
 EXTRA_DIST = \
-    heif/heif.go
+	heif/heif.go
```

Comment:
Sort of. The `test-go` target needs tabs to indent the commands to run, so to make the file consistent I changed the indentation here to use tabs, too. Otherwise we would end up having mixed tabs/spaces in a file.